tests/protocol/test_imds.py

```diff
@@ -3,7 +3,8 @@
 import json
 
 from azurelinuxagent.common.exception import HttpError
-from azurelinuxagent.common.protocol.imds import ComputeInfo, ImdsClient
+from azurelinuxagent.common.protocol.imds import ComputeInfo, ImdsClient, IMDS_IMAGE_ORIGIN_CUSTOM, \
+    IMDS_IMAGE_ORIGIN_ENDORSED, IMDS_IMAGE_ORIGIN_PLATFORM
```

Comment:
Consider refactoring the azurelinuxagent.common.protocol.imds module so it can more simply be imported, i.e.
```
import azurelinuxagent.common.protocol.imds as imds
```

Then you would reference the objects of interest as `imds.ComputeInfo`, `imds.Client`, `imds.IMAGE_ORIGIN_CUSTOM` (or possible even `imds.ImageOrigin.custom` if you further refactored the image origin information).